EtlPipelineBlobTrigger/dev-tools/process_subj_codes_csv.py

```diff
@@ -0,0 +1,29 @@
+"""
+Create json lookup files for subject code to english and subject code
```

Comment:
Is it worth adding a readme for `dev-tools` folder?

Reply:
Yes, makes sense. Done.